README.md

```diff
@@ -174,41 +174,45 @@ validates :slug, db_uniqueness: {index_name: :unique_index, case_sensitive: fals
 
 ### Problem
 
-Unfortunately, ActiveRecord's `validates_uniqueness_of` approach does not ensure 
-uniqueness. For example, we can skip validations or create two records in parallel 
+Unfortunately, ActiveRecord's `validates_uniqueness_of` approach does not ensure
+uniqueness. For example, we can skip validations or create two records in parallel
 queries. After that, our database becomes inconsistent because we assume some uniqueness
-over the table but it has duplicates.  
+over the table but it has duplicates.
 
-`validates_db_uniqueness_of` solves the problem using unique index constraints 
+`validates_db_uniqueness_of` solves the problem using unique index constraints
 in the database also providing backward compatibility with nice validations errors.
 
-### Pros and Cons
+### Advantages
 
-Advantages: 
 - Ensures uniqueness because it uses unique constraints.
-- Checks the existence of proper unique index at the boot time. 
-Use `ENV['SKIP_DB_UNIQUENESS_VALIDATOR_INDEX_CHECK'] = 'true'` 
+- Checks the existence of proper unique index at the boot time.
+Use `ENV['SKIP_DB_UNIQUENESS_VALIDATOR_INDEX_CHECK'] = 'true'`
 if you want to skip it in some cases. (For example, when you run migrations.)
-- It's two times faster in average because it skips unnecessary SQL query. See benchmarks 
-below for details.
-
-Disadvantages: 
-- Cannot handle multiple database validations at once because database raises 
-only one error per query.
+- It's two times faster in average because it skips unnecessary SQL query. See benchmarks below for details.
+- In enhanced mode if runs the query first, so for UX-critical validations we doin't have to sacrifise anything.
```

Comment:
> if runs 

it runs

> so for UX-critical validations we doin't have to sacrifise anything.

I'd reprase it to: it has different [modes]( # modes) so you can pick up the best for your needs.